src/middlewared/middlewared/plugins/zfs.py

```diff
@@ -500,57 +500,92 @@ def __init__(self, middleware):
 
                 try:
                     # FIXME: Translation
+                    human_quota_type = excess["quota_type"][0].upper() + excess["quota_type"][1:]
```

Comment:
Isn't that basically the same as `excess["quota_type"].title()`?

Reply:
You're right!

There is always a string method you didn't know about.